Categories/UIView+TCAdditions.m

```diff
@@ -129,5 +129,19 @@ - (UIView *)tc_breadthFirstSubviewThatSatisfiesBlock:(BOOL (^)(UIView *))conditi
     return nil;
 }
 
+- (void)tc_autoLayoutPinToParentViewController {
```

Comment:
Also, we should probably change the name, since it isn't actually pinning to a view controller, but rather a view